galleries/examples/subplots_axes_and_figures/secondary_axis.py

```diff
@@ -154,7 +154,7 @@ def inverse(x):
 
 ax.plot(dates, temperature)
 ax.set_ylabel(r'$T\ [^oC]$')
-plt.xticks(rotation=70)
+ax.xaxis.set_tick_params(rotation=70)
```

Comment:
I'm always unsure whether to go through `xaxis` or `ax.tick_params('x', rotation=70)`. Slight tendency for the higher-level `ax.tick_params` in simple cases but basically either is fine.